projects/lib/src/oauth-service.ts

```diff
@@ -195,7 +195,7 @@ export class OAuthService extends AuthConfig implements OnDestroy {
     }
 
     protected refreshInternal(params, noPrompt) {
-        if (this.responseType === 'code') {
+        if (!this.silentRefreshRedirectUri && this.responseType === 'code') {
```

Comment:
@KevinCathcart wouldn't it also be possible to check if there is a refreshToken available, instead of checking if the silentRefreshRedirectUri ist configured?